presto-docs/src/main/sphinx/release/release-0.238.2.rst

```diff
@@ -2,6 +2,10 @@
 Release 0.238.2
 ===============
 
+.. warning::
+
+   There is a reliability issue in this release causing frequent excessive GC events.
```

Comment:
mention that this is for zstd compression (here and also in the release note).  anyone not using zstd shouldn't be affected